eng/pipelines/common/download-specific-artifact-step.yml

```diff
@@ -11,11 +11,11 @@ steps:
     displayName: 'Download specific ${{ parameters.displayName }}'
     inputs:
       buildType: specific
-      project:  'internal' # 'internal' or 'public'
+      project:  'public' # 'internal' or 'public'
       pipeline: 'Antigen'
       buildVersionToDownload: specific
-      branchName: 'kpathak/antigen-ci'
-      buildId: '1385069'
+      branchName: 'refs/pull/59376/head'
```

Comment:
I don't understand why this is the branch name that is being used, isn't this an unrelated PR?

Reply:
This file is added for testing purpose to download artifacts from previous `buildId` instead of waiting to build coreclr/libraries. This is handy to do quick testing during pipeline development. It is not used in main flow.